x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/indexlifecycle/LifecyclePolicyTests.java

```diff
@@ -138,7 +138,7 @@ public static LifecyclePolicy randomTestLifecyclePolicy(@Nullable String lifecyc
                 MockAction action = new MockAction();
                 actions.put(action.getWriteableName(), action);
             }
-            String phaseName = randomAlphaOfLength(10);
+            String phaseName = randomFrom(TimeseriesLifecycleType.VALID_PHASES);
```

Comment:
is this required?

Reply:
I don't *think* so any more, I'll revert